pkgs/by-name/ov/overlayed/webui.nix

```diff
@@ -0,0 +1,43 @@
+{
+  src,
+  meta,
+  version,
+  lib,
+  stdenv,
+  nodejs,
+  pnpm,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  inherit version src meta;
+  pname = "overlayed-webui";
+
+  pnpmDeps = pnpm.fetchDeps {
+    src = "${finalAttrs.src}/apps/desktop";
+    version = finalAttrs.version;
+    pname = finalAttrs.pname;
+    hash = "sha256-FBJrVXTgKh8Op7TiZjA/i2LY2OOIKmqGR+GwGUecqVI=";
+  };
+
+  nativeBuildInputs = [
+    nodejs
+    pnpm.configHook
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    cd apps/desktop
+    node --max_old_space_size=1024000 ./node_modules/vite/bin/vite.js build
```

Comment:
Is there something else you could do here, e.g. `pnpm build`?
How important is `--max_old_space_size=1024000`?